### ISSUE TITLE
metadata editor form allows improper contributor url

### ISSUE THREAD

**Issue by bendichter** [dandi/dandi-archive#1477]
<img width="926" alt="image" src="https://user-images.githubusercontent.com/844306/217292426-8ba9fd53-cdb4-4e93-8dbb-98582da0374e.png">

This form is allowed in the metadata editor, but fails validation because it is missing http://. Can we modify the form to validate the URL field and prevent users from entering improper urls?

**Comment by bendichter** [comment]
To make matters worse, the validation error in the web interface is not very informative

<img width="879" alt="image" src="https://user-images.githubusercontent.com/844306/217293055-c61ea766-e8ba-430d-88af-1349e976a2b8.png">

**Comment by satra** [comment]
@mvandenburgh - are we using ajv with vjsf? that may help with some of these. i can also look into creating a pyscript runnable version of the schema. since vjsf allows external validators.

**Comment by mvandenburgh** [comment]
> @mvandenburgh - are we using ajv with vjsf? that may help with some of these. i can also look into creating a pyscript runnable version of the schema. since vjsf allows external validators.

No, but I believe vjsf has built-in support for ajv, so we could.

**Comment by mvandenburgh** [comment]
Opened #1601 to track the AJV integration task explicitly.
